documentation/homework/week1/todo-app/src/components/todo-item.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+
+export default class TodoItem extends Component {
+  render() {
+    const { description, deadLineDate, isDone} = {
+      description: this.props.description,
+      deadLineDate: new Date(this.props.deadLineDate).toDateString(),
+      _isDone: this.props.isDone, // this is a privte property just like state
```

Comment:
What is your reasoning for setting a prop to a private class property? It's already inside `this.props`. Usually, underscored items are "private" in a different sense - they are internals for libraries which are meant to be ignored by developers etc. Unless you have a very good reason to do this, I would highly recommend you access `this.props.isDone` directly instead of doing this.